src/ts/Offers/api.ts

```diff
@@ -1,7 +1,7 @@
 import API from '../API';
 
 export function get() {
-  return fetch(API.offer, {
+  return window['fetch'](API.offer, {
```

Comment:
I added here `window` because testing environment doesn't support types for `fetch` (`unable to compile TypeScript`). I can install types for `fetch` (like `@types/whatwg-fetch`) but then I get errors on frontend (double type declaration).